### PR TITLE
Added pagination for compete / practice results.

### PR DESCRIPTION
**Pull request by deyordanov**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/1431

### CODE REVIEW

File: Services/Common/OJS.Services.Common.Models/Contests/Results/ContestResultsViewModel.cs [hunk 1]
```diff
@@ -31,6 +31,10 @@ public class ContestResultsViewModel
 
     public bool UserHasContestRights { get; set; }
 
+    public int ItemsPerPage { get; set; }
+
+    public int PagesCount { get; set; }
```
Comment (gogo4ds) [inline Services/Common/OJS.Services.Common.Models/Contests/Results/ContestResultsViewModel.cs:36]: There is `PagedResults` property, why page info is needed here?

File: Services/Common/OJS.Services.Common/Implementations/ContestResultsAggregatorCommonService.cs [hunk 2]
```diff
@@ -56,13 +57,17 @@ public ContestResultsViewModel GetContestResults(ContestResultsModel contestResu
                                              contestResultsModel.Official)
                                          .Count();
 
+        contestResults.PagesCount = totalParticipantsCount > contestResultsModel.ItemsPerPage
+            ? (int)Math.Ceiling((double)totalParticipantsCount / contestResultsModel.ItemsPerPage)
+            : 1;
```
Comment (gogo4ds) [inline Services/Common/OJS.Services.Common/Implementations/ContestResultsAggregatorCommonService.cs:62]: Can we use `ToPagedResultAsync` from `QueryableExtensions`?

**Review by gogo4ds — CHANGES_REQUESTED** [review]

File: Services/Common/OJS.Services.Common.Models/Contests/Results/ContestResultsViewModel.cs (hunk 1 above)
Comment (deyordanov) [reply Services/Common/OJS.Services.Common.Models/Contests/Results/ContestResultsViewModel.cs]: Done.

File: Services/Common/OJS.Services.Common/Implementations/ContestResultsAggregatorCommonService.cs (hunk 2 above)
Comment (deyordanov) [reply Services/Common/OJS.Services.Common/Implementations/ContestResultsAggregatorCommonService.cs]: Done.

**Review by gogo4ds — APPROVED** [review]

**Pull request merged**